arm-dependencies.sls

```diff
@@ -0,0 +1,91 @@
+{% from 'common/map.jinja' import config %}
+
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+{% set path = 'https://servo-rust.s3.amazonaws.com/ARM' %}
```

Comment:
`path` is a little ambiguous. How about `rootfs_repo` (feel free to put something else if you can think of something better)?